src/NHSD.BuyingCatalogue.Ordering.Api/Services/CompleteOrder/CompleteOrderService.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace NHSD.BuyingCatalogue.Ordering.Api.Services.CompleteOrder
 {
-    internal sealed class CompleteOrderService : ICompleteOrderService
+    public sealed class CompleteOrderService : ICompleteOrderService
```

Comment:
Why has this been changed from `internal` to `public`?

Reply:
I was getting error during implementaion but resolved with changes in *.csproj changes, so reverting back to internal